src/ocnicepost.fd/init_mod.F90

```diff
@@ -106,6 +106,7 @@ subroutine readnml
     if (nxr == 1440 .and. nyr == 721) fdst = '0p25'      ! 1/4deg rectilinear
     if (nxr == 720  .and. nyr == 361) fdst = '0p5'       ! 1/2 deg rectilinear
     if (nxr == 360  .and. nyr == 181) fdst = '1p0'       ! 1 deg rectilinear
+    if (nxr == 72   .and. nyr == 36) fdst = '5p0'        ! 5 deg rectilinear
```

Comment:
Apologies @DeniseWorthen 
Can we for the sake of consistency, make this `5p00` and the one above as `0p50` and `0p10`? 
Does that change impact elsewhere?

Reply:
It would require a change to the cpld_gridgen utility, because that is the name tag that it uses when creating the esmf weights file. They were named that way originally because that is the name-style I was given as the requirement. You could of course link the name locally, but I realize that is not ideal.

Reply:
ok. disregard this suggestion.